docs/src/main/asciidoc/smallrye-metrics.adoc

```diff
@@ -3,77 +3,75 @@ This guide is maintained in the main Quarkus repository
+IMPORTANT: link:https://quarkus.io/guides/micrometer[Micrometer] is the recommended approach to metrics for Quarkus. Use the SmallRye Metrics extension when it is required to retain MicroProfile specification compatibility.
 
-== Prerequisites
+.Prerequisites
 
-To complete this guide, you need:
+* Approximately 15 minutes.
+* An IDE.
+* JDK 11+ installed with `JAVA_HOME` configured appropriately. This could be tested using the `java -version` and `javac` commands, after the execution of which a user with correctly setup Java obtains information about the current version and the possible `javac` options of usage.
+* Apache Maven {maven-version} installed.
 
-* less than 15 minutes
-* an IDE
-* JDK 11+ installed with `JAVA_HOME` configured appropriately
-* Apache Maven {maven-version}
+.Architecture
 
-== Architecture
-In this example, we build a very simple microservice which offers one REST endpoint and that endpoint serves
-for determining whether a number is prime. The implementation class is annotated with some metric annotations
-so that while responding to user's requests, some metrics are gathered. The meaning of each metric will be explained later.
+In this example, we build a very simple microservice that offers one REST endpoint. This endpoint serves for determining whether a number is prime. The implementation class is annotated with certain metric annotations so that while responding to users' requests, certain metrics are gathered. The meaning of each metric is explained later.
 
-== Solution
+.Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+We recommend that you follow the instructions in the next sections and create the application step by step. However, you can skip to the completed example.
 
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+. Clone the Git repository:
++
+[source,bash,subs=attributes+]
+----
+$ git clone {quickstarts-clone-url}
+----
 
-The solution is located in the `microprofile-metrics-quickstart` {quickstarts-tree-url}/microprofile-metrics-quickstart[directory].
+* Alternatively download a {quickstarts-archive-url}[Quickstarts archive]. The solution is located in the `microprofile-metrics-quickstart` {quickstarts-tree-url}/microprofile-metrics-quickstart[directory].
 
-== Creating the Maven Project
+* xref:creating-a-maven-project_{context}[]
+* xref:writing-an-application_{context}[]
+* xref:running-and-using-the-application_{context}[]
 
-First, we need a new project. Create a new project with the following command:
+[id="creating-a-maven-project_{context}"]
+== Creating a Maven project
 
+. To create a new Maven project:
++
 [source,bash,subs=attributes+]
 ----
-mvn io.quarkus.platform:quarkus-maven-plugin:{quarkus-version}:create \
+$ mvn io.quarkus.platform:quarkus-maven-plugin:{quarkus-version}:create \
```

Comment:
Please no extra `$` in commands, unless we can somehow do it in a way that it won't appear in the clipboard when the user performs ctrl+C over the contents of the box. 
And that is again something that would need to be applied to all guides (for consistency), not just this one